libopenage/util/fds.h

```diff
@@ -20,7 +19,7 @@ class FD {
 	/**
 	 * wraps an existing FD
 	 */
-	FD(int fd=STDOUT_FILENO);
+	FD(int fd);
```

Comment:
Not really. Just with `STDOUT_FILENO`. Usually, it's defined in `unistd.h` which is just absent for VS. Turned out `#define STDOUT_FILENO 1` was too much to ask for.